persistence/msmq/index_timeouts_msmqtransport_[2,).partial.md

```diff
@@ -0,0 +1,13 @@
+---
+component: MSMQPersistence
+---
+
+## Timeouts persistence
+
+`MsmqPersistence` provides persistence only for storing event subscriptions. By default, NServiceBus also requires a timeout persistence, which is used by [delayed retries](/nservicebus/recoverability/#delayed-retries), [saga timeouts](/nservicebus/sagas/timeouts.md) and for [delayed delivery](/nservicebus/messaging/delayed-delivery.md).
+
+If none of these features are used, timeouts can be disabled:
```

Comment:
This used to be followed by a snippet. Should that be here or should this sentence be removed?

Reply:
Given how the new delayed delivery (via SQL) works, we can't just disable it as a feature, as it is done through transport configuration (`MsmqTransport.DelayedDelivery`). 

Should we link or add a snippet like [this](https://docs.particular.net/transports/msmq/delayed-delivery?version=msmqtransport_2), here?

Reply:
Updated